src/components/ProductInfo.vue

```diff
@@ -0,0 +1,392 @@
+
+      // Remove minus class on all other buttons
+      contextPanelGroup.find('.accordion-plus').not(contextButton).removeClass('accordion-minus');
+    },
+  },
+
+  computed: {
+    loading() {
+      return this.$apollo.queries.product.loading;
+    },
+
+    empty() {
+      return !Object.keys(this.product).length;
+    },
+
+    currentProduct() {
+      return this.product.masterData.current || {};
+    },
+
+    matchingVariant() {
+      return this.currentProduct.variant || {};
+    },
+
+    hasPrice() {
+      return this.matchingVariant.price;
+    },
+
+    hasDiscount() {
+      return this.matchingVariant.price.discounted;
+    },
+
+    discountedPrice() {
+      return this.matchingVariant.price.discounted.value;
+    },
+
+    originalPrice() {
+      return this.matchingVariant.price.value;
+    },
+
+    images() {
+      return this.matchingVariant.images;
+    },
+
+    productAttributes() {
+      return this.matchingVariant.attributes;
+    },
+  },
+
+  apollo: {
+    product: {
+      query: gql`
+        query Product($locale: Locale!, $sku: String!, $currency: Currency!) {
+          product(sku: $sku) {
+            id
+            masterData {
+              current {
+                name(locale: $locale)
+                slug(locale: $locale)
+                variant(sku: $sku) {
+                  sku
+                  attributes {
+                    ...on mainProductType {
+                      designer {
+                        label
+                        key
+                        name
+                      }
+                      colorFreeDefinition {
+                        value(locale: $locale)
+                        name
+                      }
+                      size{
+                        value
+                        name
+                      }
+                      style{
+                        key
+                        label
+                        name
+                      }
+                      gender{
+                        key
+                        label
+                        name
+                      }
+                      articleNumberManufacturer{
+                        name
+                        value
+                      }
+                    }
+                  }
+                  images {
+                    url
+                  }
+                  price(currency: $currency) {
+                    value {
+                      ...printPrice
+                    }
+                    discounted {
+                      value {
+                       ...printPrice
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        fragment printPrice on BaseMoney {
+          centAmount
+          fractionDigits
+        }`,
+      variables() {
+        return {
+          locale: this.$i18n.locale,
+          currency: 'EUR',
```

Comment:
This shouldn't be hardcoded.

Reply:
I temporarily copied number formats object from i18n file locally to price mixin file so we can use it until we find a solution for the issue after I create it.